sdk/python/lib/pulumi/runtime/invoke.py

```diff
@@ -194,7 +196,7 @@ async def invoke_async(
 def _invoke(
     tok: str,
     props: "Inputs",
-    opts: Optional[InvokeOptions],
+    opts: Optional[Union[InvokeOptions, InvokeOutputOptions]],
```

Comment:
Do we need this?

Reply:
I think so, this is an internal function, but it gets called from the public direct and output form invoke functions.